cli-script.php

```diff
@@ -82,23 +82,20 @@
 // Connect to database
 if(!isset($databaseConfig) || !isset($databaseConfig['database']) || !$databaseConfig['database']) {
 	echo "\nPlease configure your database connection details.  You can do this by creating a file
-called _ss_environment.php in either of the following locations:\n\n";
-	echo " - " .  BASE_PATH  . DIRECTORY_SEPARATOR . "_ss_environment.php\n - ";
-	echo dirname(BASE_PATH) . DIRECTORY_SEPARATOR . "_ss_environment.php\n\n";
+called .env in " . BASE_PATH;
```

Comment:
Can we support .env files that aren't in the PATH, please?

 * I have a `~/Sites` folder with 50-odd projects and a single `_ss_environment.php` file. This is a helpful use case
 * We have a lot of sites where `BASE_PATH` is `/sites/something/www` and `_ss_environment.php` is in `/sites/something`. There are a number of benefits to this approach and I wouldn't want to make it impossible.